src/dotnet/Fable.Core/Import/Node/Path.fs

```diff
@@ -0,0 +1,49 @@
+module rec Fable.Import.Node.Path
+
+open Fable.Core
+
+[<AutoOpen>]
+module path_types = 
+    type PathObjectProps = 
+        abstract dir : string
+        abstract root : string
+        [<Emit("$0.base{{=$1}}")>] // using this because back ticks are way too ugly to read in code
+        abstract baseName : string
+        abstract name : string
+        abstract ext : string
```

Comment:
If we want to use `createEmpty<PathObjectProps>` with this, we will have to make all the fields mutable, that is adding ` with get, set` and the end of every field.